uniffi/src/interface/mod.rs

```diff
@@ -96,6 +99,10 @@ impl<'ci> ComponentInterface {
         }
         // Now that the high-level API is settled, we can derive the low-level FFI.
         ci.derive_ffi_funcs()?;
+        ci.errors.push(Error {
+            name: GENERIC_RUST_ERROR.to_string(),
+            values: vec!["UnknownError".to_string()],
```

Comment:
Note to self to maybe add `Panic` as a variant here and look at the actual conversions that `ffi_support` covers.. I might have this as a part of #32 But seems more relevant to this PR... We'll see